x-pack/plugins/observability_solution/dataset_quality/common/constants.ts

```diff
@@ -19,3 +19,10 @@ export const NONE = 'none';
 
 export const DEFAULT_TIME_RANGE = { from: 'now-24h', to: 'now' };
 export const DEFAULT_DEGRADED_DOCS = { percentage: 0, count: 0 };
+
+export const SO_SEARCH_LIMIT = 10000;
+export const PACKAGES_SAVED_OBJECT_TYPE = 'epm-packages';
+export const DASHBOARD_SAVED_OBJECT_TYPE = 'dashboard';
+
+export const MANAGEMENT_APP_LOCATOR = 'MANAGEMENT_APP_LOCATOR';
```

Comment:
is this similar to what we have in `packages/deeplinks/management/deep_links.ts`?

Reply:
I feel like declaring this constant doesn't belong to our plugin, is this one defined anywhere else?